lld/ELF/Driver.cpp

```diff
@@ -1487,6 +1487,7 @@ static void readConfigs(Ctx &ctx, opt::InputArgList &args) {
       args, "keep-text-section-prefix", "nokeep-text-section-prefix", false);
   ctx.arg.zLrodataAfterBss =
       getZFlag(args, "lrodata-after-bss", "nolrodata-after-bss", false);
+  ctx.arg.zNoBtCfi = hasZOption(args, "nobtcfi");
```

Comment:
Initialize the variable to false. Move this to setConfigs and only do it when osabi is openbsd.

Reply:
I am not sure what you mean by initialize the variable to false.

Reply:
Also the comment at the top of setConfigs() doesn't really align with this being added there..
```
// Some Config members do not directly correspond to any particular
// command line options, but computed based on other Config values.
```

Reply:
This comment will be stale and can be updated.

I mean `bool zNoBtCfi = false;` in Config.h